docs/docs/explore/project-chat.md

```diff
@@ -0,0 +1,169 @@
+
+<img src='/img/explore/chat/project-chat.png' class='rounded-gif'/>
+<br />
+
+## How It Works
+
+AI Chat uses the same [Rill MCP Server](/explore/mcp) technology that powers external AI integrations with tools like Claude Desktop. This means:
+
+- **Governed Data Access** - The AI only queries [metrics views](/build/metrics-view) you've already defined, ensuring accuracy and consistency
+- **Structured Responses** - Results are based on predefined measures and dimensions, not raw database queries
+- **Context-Aware** - The AI understands your metrics view structure, including available dimensions, measures, and time ranges
+- **Secure** - Respects your [project's access](/build/metrics-view/security) controls and user permissions
+
+## Accessing AI Chat
+
+### Access AI Chat from Project Home
+
+1. Navigate to your [Rill Cloud](https://ui.rilldata.com) project home page
+2. Click on the **AI** tab in the project navigation
+3. Start typing your question in the chat interface
+
+### Access AI Chat from a Dashboard
+
+You can also access AI Chat directly while exploring a dashboard, making it easy to ask questions about what you're currently viewing:
+
+1. While viewing any [Explore dashboard](/explore/dashboard-101), look for the **AI Chat icon** in the top navigation bar
+2. Click the AI Chat icon to open the chat panel alongside your dashboard
+3. Ask questions about the data you're currently viewing
+<img src='/img/explore/chat/dashboard-chat.png' class='rounded-gif'/>
+<br />
+
+
+When you open AI Chat from a dashboard, the AI is automatically aware of:
+- **Current dashboard context** - The metrics view you're viewing
+- **Applied filters** - Any dimension or measure filters you've set
+- **Time range** - The time period currently selected
+- **Comparison settings** - Any active time comparisons
+
+This context-aware functionality means you can ask questions like:
+- "Why did this metric spike?" (referring to what's visible on screen)
+- "What's driving this change?" (analyzing the current time period)
+- "Show me more details about these results" (diving deeper into filtered data)
+
+:::tip Context-Aware Queries
+Opening AI Chat from within a dashboard allows for more natural, context-aware questions. The AI understands what you're looking at, so you don't need to repeat filters or time ranges in your questions.
+:::
+
+## Understanding Responses
+
+AI Chat provides rich, multi-layered responses to help you understand your data quickly while maintaining easy access to deeper exploration:
+
+### What's Included in Responses
+
+1. **Summary** - A concise answer to your question with key findings and insights
+2. **Canvas Dashboard Components** - Interactive visualizations built using canvas dashboard widgets that help you see patterns at a glance. These can include:
```

Comment:
If we publish these docs this week, we'll want to tag this with a "coming soon" label